packages/terra-table/tests/wdio/multi-select-table-spec.js

```diff
@@ -1,29 +1,27 @@
-const viewports = Terra.viewports('medium');
```

Comment:
why did the above screenshots change?

Reply:
There was a `beforeEach` hook  that brought focus to the top of the table between every test. One of the tests was supposed to focus each row but was not able to because of the `beforeEach` hook. I think some of the screenshots hadn't been updated since the focus glow became yellow.

Reply:
I guess it's the table screenshots from the single select table

Reply:
and i see your response below